src/main/kotlin/caculator/Calculator.kt

```diff
@@ -0,0 +1,24 @@
+package caculator
+
+class Calculator(private val delimiter: String) {
+
+    constructor() : this(" ")
+
+    fun calculate(text: String): Number {
+        if (text.trim().isEmpty()) {
+            throw IllegalArgumentException("text must be not empty")
+        }
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(text.trim().isNotEmpty()) { "text must be not empty" }
```